packages/grant-explorer/src/features/collections/community/index.ts

```diff
@@ -0,0 +1,36 @@
+export type CommunityCollection = {
```

Comment:
@owocki this is what want to update

Reply:
updated! b843af55b4f8b257a1b45d31464c1d9dc74ad3b1